classes/Animal.py

```diff
@@ -0,0 +1,5 @@
+class Animal:
+    def __init__(self, name: str, speed: int, step_interval=1):
```

Comment:
it's not clear from the name what the difference between speed and step insterval is